packages/fast-components-react-msft/src/action-toggle/README.md

```diff
@@ -0,0 +1,13 @@
+# Action Toggle
```

Comment:
Please add one line break between headings and paragraphs in markdown. (this rule applies to all markdown 'elements').